### DISCUSSION TITLE
1.36.1

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#22023] [General]
Feel free to discuss the [v1.36.1](https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.36.1) release here!

**Comment by mundschenk-at** [comment]
@Koenkk I think there is something wrong with the auto-changelog.

**Reply by Koenkk** [reply]
Which one?

**Reply by mundschenk-at** [reply]
It includes a lot of things from previous releases? E.g. `origin` and `scenes` discovery.

**Reply by Koenkk** [reply]
Ah, you mean [this one](https://gist.github.com/Koenkk/bfd4c3d1725a2cccacc11d6ba51008ba). It's indeed broken until the next commit to `dev`

**Reply by mundschenk-at** [reply]
No I meant the one in the 1.36.1 release announcement.

**Reply by Minnozz** [reply]
Can confirm, the GitHub changelog on the new release shows the experimental Ember adapter being added, but that was in the previous release

**Reply by Koenkk** [reply]
Whoops, fixed it now.

**Comment by burmistrzak** [comment]
@Koenkk Happy release day! 💪
Great to have my improvements for the Twinguard already included, but I'll definitely have to rework that AQI <> TVOC conversion. It's unfortunately not a linear scale... 😒

**Comment by wurmi-007** [comment]
Hallo herzlichen Dank für die neue Version, kann es sein das das Dashboard nicht mehr funktioniert. Beim anklicken eines Schalters schaut es optisch aus als wäre alles ok aber beim Schalter tut sich nichts???

**Comment by wurmi-007** [comment]
Hello, thank you very much for the new version, it may be that the dashboard no longer works. When you click on a switch it looks like everything is ok but nothing happens with the switch???

**Reply by francisp2** [reply]
Previously my router-flashed Sonoff ZBBridge was supported with this external convertor
```
const exposes = require('zigbee-herdsman-converters/lib/exposes');
const fz = {...require('zigbee-herdsman-converters/converters/fromZigbee'), legacy: require('zigbee-herdsman-converters/lib/legacy').fromZigbee};
const tz = require('zigbee-herdsman-converters/converters/toZigbee');
const reporting = require('zigbee-herdsman-converters/lib/reporting');
const extend = require('zigbee-herdsman-converters/lib/extend');
const e = exposes.presets;

module.exports = [
    {
        zigbeeModel: ['UT-02'],
        model: 'EFR32MG21',
        vendor: 'Custom devices (DiY)',
        description: 'EFR32MG21 Silabs Router',
        fromZigbee: [],
        toZigbee: [],
        exposes: [],
    },
    {
        zigbeeModel: ['UT-01'],
        model: 'EFR32MG21',
        vendor: 'Custom devices (DiY)',
        description: 'EFR32MG21 Zigbee Bridge Router',
        fromZigbee: [],
        toZigbee: [],
        exposes: [],
    },
];
```
but in 1.36.1 it is unsupported.

**Reply by mrskycriper** [reply]
@francisp2 There was a breaking change in zigbee herdsman converters 19.0.0, legacy extend no longer exists. But since it was used in external converter template, it is widely imported in a lot of them without real use.

Just remove the `const extend = require('zigbee-herdsman-converters/lib/extend');` line and it will work just fine again.

**Comment by cavolo** [comment]
Hi, I try new release but have a problem with my power counter. After update the variables name changed and loose all hystory data.
For this counter use an external conversion.
I back to version 1.36.0

**Reply by wurmi-007** [reply]
Everything is known, dashboard does not function, binding is faulty!

**Comment by jamesonuk** [comment]
Seems my Hue Wall Switch modules (https://www.zigbee2mqtt.io/devices/929003017102.html#philips-929003017102) all reconfigured after installing 1.36.1 which then bound the Hue specific cluster and broke my lights.....
Took me a while to figure out what had happened as reverted back to 1.36 and 1.35 but things still weren't working.   I then removed the Hue specific cluster and they worked and when I upgraded to 1.36.1 again they stopped working and Hue specific cluster was once again bound

**Reply by oneOfFiveAnd** [reply]
I also have a problem with hue wall switch modules. Didn't have the time to try solving yet, here is the most obvious error i can see. it seems that if they are configured as single push button, they now generate two actions, both with payload e.g. "left_press" with a single button push

Having said that, thanks for maintaining such a fantastic integration ... not everything can go right all the time

**Reply by oneOfFiveAnd** [reply]
addition. i have wall switches configured as due push buttons. After the version upgrade, only one of the buttons creates an activity when pushed, the other one has no effect any more. Multiple switches with same problem.

**Reply by bjornjonsson** [reply]
My wall switch modules also stopped working after the update. I use mine in a single push button configuration bound to a group. I could get them working again in 1.36.1 by:

1. Reconfigure the wall module as single push button. The update had set them back to single rocker.
2. Unbind OnOff and manuSpecificPhilips from the Coordinator. I only had PowerCfg bound to the Coordinator after this.
3. Bind OnOff and LeverCtrl to the group.

The wall switch seemed to function after the second step, so I'm not sure if the third step is necessary.  I did it anyway since that is what the documentation says.

**Comment by weljajoh** [comment]
After upgrading to 1.36.1 all my Hue dimmers bound to a group no longer worked for turning on and off the lights in the group. They seem to have grown a new binding from source endpoint 1  to coordinator destination endpoint 1 during the upgrade. After removing that binding, the dimmers work again for turning lights on and off.
<img width="1569" alt="Screenshot 2024-04-02 at 20 35 08" src="https://github.com/Koenkk/zigbee2mqtt/assets/107270490/25aa3960-95f3-4772-a27d-64c7d4a368f0">

**Reply by JohannCR** [reply]
Can you explain how you did it ? When I try to unbind it gives me an error... But since I never used binding, not sure I did it right ^^ thx

**Reply by weljajoh** [reply]
The device needs to be awake for the unbind to succeed, so first press a button on the dimmer itself to wake it up and then click the unbind button in the Z2M GUI.

**Reply by JohannCR** [reply]
Thanks it works. Got other devices with the same coordinator bind like an Aqara SSM-U01 switch but unbinding does not work even when I hit the switch before and after... Any tip for this one ?

**Reply by JohannCR** [reply]
FYI tried this that didn't work either (please do tell if that's not correct, thanks^^):
service: mqtt.publish
data:
  topic: zigbee2mqtt/bridge/request/device/unbind
  payload: >-
    {"clusters":["genOnOff","genDeviceTempCfg"], "from": "Inter Petit Bureau",
    "to": "Coordinator"}

**Reply by JohannCR** [reply]
Nevermind, pairing again worked

**Comment by ghost** [comment]
Same what wurmi-007 has noted: dashboard does not function like in 1.36. Switches/lights cannot be controlled like in 1.36. Going to "devices", selecting the device and going to "exposes" lets control switches and lights. (on/off, dimming, changing colors, etc.).  Just not in dashboard.

Nevertheless, many thanks for releasing 1.36.1. Very happy that the IKEA Water Leak Detector is supported now!

**Reply by MMi1603** [reply]
Same in my case. All device switches work as externally programmed (in my case openhab) or in the detailed device options, but not in the dashboard. When switching a device it does not show up the change in the dashboard. Strange that LQI, contacts or "last seen" are ok there, only the switches (plugs or lights) do not work.

I had used converters before (namely for the Ikea Badring), but deleted the according line in the configuration.yaml and my js-files . Maybe there is another file referring to a bad "const extend=..." as described by  @mrskycriper above.

Regards,

MMi

**Comment by kamilnadzieja** [comment]
Never had any issues with the updates but here I had to revert back to 1.36.0. There is something wrong with the external converters (like they are not imported at all to z2m) plus I had many issues with the ikea devices (they were appearing as online but no data were going through). But I just want to say big thank you for your hard work and efforts!

Edit  - wrong, not ikea at most but aqara/xiaomi - 70% of them went offline.

**Comment by mrskycriper** [comment]
# PSA about broken external converters

There was a breaking change in zigbee herdsman converters 19.0.0, legacy extend no longer exists. But since it was used in external converter template, it is widely imported in a lot of them without real use.

Just remove the `const extend = require('zigbee-herdsman-converters/lib/extend');` line from the file and it will work just fine again.

**Reply by mrskycriper** [reply]
@Koenkk Perhaps there should have been a "Breaking changes" section about the legacy extend in the changelog. There are a lot of broken external converters right now.

**Reply by cavolo** [reply]
Thank you so much! I remove the line and work perfectly.

**Reply by Koenkk** [reply]
I will add a note to the release notes.

**Reply by derkorte** [reply]
In my case there's a converter, which does not work by removing the extend line. Any ideas? 
The device is shown as not supported. This worked perfekt since 2 years till last update.

const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
const tz = require('zigbee-herdsman-converters/converters/toZigbee');
const exposes = require('zigbee-herdsman-converters/lib/exposes');
const reporting = require('zigbee-herdsman-converters/lib/reporting');
const e = exposes.presets;
const ea = exposes.access;

const definition = {
    zigbeeModel: ['RGBWW Lighting'], // The model ID from: Device with modelID 'lumi.sens' is not supported.
    model: '511.050', // Vendor model number, look on the device for a model number
    vendor: 'Sunricher',
    description: 'Zigbee 3.0 LED controller for 5in1 RGB+CCT LEDs',
    extend: extend.light_onoff_brightness_colortemp_color({colorTempRange: [155, 450]}),
};

module.exports = definition;

**Reply by mrskycriper** [reply]
> In my case there's a converter, which does not work by removing the extend line. Any ideas? 
> 
> The device is shown as not supported. This worked perfekt since 2 years till last update.
> 

@derkorte First of all - please submit known working external converters for devices into the project. This will allow seamless updates in the future. External converters are not recommend for long term use as they will inevitably break without maintenance.

As for this converter, you will need to replace the legacy extend itself. Use `light` modern extend as a replacement.

**Reply by derkorte** [reply]
@mrskycriper how does the line light modern extend has to look like? I've tried a few variations but it still not work

**Reply by cbrunnkvist** [reply]
@derkorte `const light = require('zigbee-herdsman-converters/lib/light');`? Not an authoritative answer; just judging from the directory structure :)

**Reply by Nerivec** [reply]
https://github.com/search?q=repo%3AKoenkk%2Fzigbee-herdsman-converters%20%27511.050%27&type=code

**Reply by derkorte** [reply]
@Nerivec thanks, but this still not work. In my old converter this line still exist. I've replaced it with this one but device is still unsupported.

@mrskycriper can you give a hint to replace the extend line?
I thought that devices listed in devices from zigbee herdsman where still known... but i will contribute them in future.

**Reply by mrskycriper** [reply]
@derkorte Can you provide some info on this device? Is it made by Sunricher? What model is printed on it? I'd like to just add it as a supported device.

As for **TEMPORARY** converter, you can use this:

```js
const {light} = require('zigbee-herdsman-converters/lib/modernExtend');

const definition = {
    zigbeeModel: ['RGBWW Lighting'],
    model: '511.050',
    vendor: 'Sunricher',
    description: 'Zigbee 3.0 LED controller for 5in1 RGB+CCT LEDs',
    extend: [
        light({colorTemp: {range: [155, 450]}, color: true}),
    ],
};

module.exports = definition;
```

**Reply by derkorte** [reply]
It's the iluminize 511.050 led driver but with fingerprint Sunricher.

The converter above work perfect. Thank you!

**Reply by mrskycriper** [reply]
> It's the iluminize 511.050 led driver but with fingerprint Sunricher.

So, it's a Iluminize device with the Sunricher branding on it? Can you please share photos of it? What is the model printed on the device itself?

**Reply by derkorte** [reply]
![IMG_20240408_191038](https://github.com/Koenkk/zigbee2mqtt/assets/99035078/4b532133-12cd-4a5a-803d-2e3f828ef426)
![Screenshot_2024-04-08-19-11-39-648_io homeassistant companion android](https://github.com/Koenkk/zigbee2mqtt/assets/99035078/da52910c-98b9-4396-b9dc-bc2298afc276)

**Reply by derkorte** [reply]
Should I create a pull request or will it be updated by someone?

**Reply by mrskycriper** [reply]
> Should I create a pull request or will it be updated by someone?

I'll add it

**Reply by Nerivec** [reply]
@mrskycriper From what I glanced, several of iluminize devices have [number based or letter/text based](https://github.com/Koenkk/zigbee-herdsman-converters/blob/00b10b7cf082eb3438f76f094970c6739f969853/src/devices/iluminize.ts#L130). Might be the same thing as this with others (some currently only have text based, others only number based). Worth making a note of it 😉

**Reply by mrskycriper** [reply]
@Nerivec I'll just add this model to exiting definition for `511.050`. Seems to be the same exact thing, maybe just another revision.

**Reply by cbrunnkvist** [reply]
These docs will need to be updated too: [Support new devices: Extending the external definition](https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html#_2-1-extending-the-external-definition)

**Reply by mrskycriper** [reply]
> These docs will need to be updated too: [Support new devices: Extending the external definition](https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html#_2-1-extending-the-external-definition)

They were updated recently to remove all mentions of legacy extend

**Comment by CinciRyan** [comment]
Hello!  I had to roll back to 1.36.0-1 because I lost color control of all of my Sengled E21-N1EA bulbs.  In Z2M I had no option to change colors (the Color (X,Y) Field under 'Exposes'. Rolling back brings that back.  https://www.zigbee2mqtt.io/devices/E21-N1EA.html#sengled-e21-n1ea
<img width="1034" alt="Screenshot 2024-04-03 at 2 42 57 PM" src="https://github.com/Koenkk/zigbee2mqtt/assets/47645423/c2d3c46f-73e9-4566-8983-514ab9d5aceb">

**Reply by CinciRyan** [reply]
Here's the same model bulb on a different instance running 1.36.1
<img width="1034" alt="Screenshot 2024-04-03 at 2 45 46 PM" src="https://github.com/Koenkk/zigbee2mqtt/assets/47645423/d8c17568-03a6-4e5f-8e71-885a693d3cf0">

**Reply by mrskycriper** [reply]
Fixed this in https://github.com/Koenkk/zigbee-herdsman-converters/pull/7317 ASAP

**Comment by GustavoScramin** [comment]
Hello everyone, I was using this converter below for my lamp and fan, it is the same device. But with this latest z2mqtt update, my light bulb just disappeared, and I think z2mqtt is referencing another device. Does anyone know how I can add the lamp back? I already deleted the line of code "const extend = require('zigbee-herdsman-converters/lib/extend');"

`const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
const tz = require('zigbee-herdsman-converters/converters/toZigbee');
const exposes = require('zigbee-herdsman-converters/lib/exposes');
const reporting = require('zigbee-herdsman-converters/lib/reporting');
const extend = require('zigbee-herdsman-converters/lib/extend');
const e = exposes.presets;
const ea = exposes.access;
const tuya = require('zigbee-herdsman-converters/lib/tuya');

const definition = {
    fingerprint: [
        {
            modelID: 'TS0601',
            manufacturerName: '_TZE200_lawxy9e2',
        },
    ],
    model: 'TS0601_fan_and_light_switch',
    vendor: 'TuYa',
    description: 'Fan & light switch',
    fromZigbee: [tuya.fz.datapoints],
    toZigbee: [tuya.tz.datapoints],
    onEvent: tuya.onEventSetTime,
    configure: tuya.configureMagicPacket,
    exposes: [
	e.binary('status_indication', ea.STATE_SET, 'ON', 'OFF').withDescription('Light switch'),
        tuya.exposes.switch(),
        tuya.exposes.countdown().withValueMin(0).withValueMax(720).withUnit('m').withDescription('Max ON time in minutes'),
        e.power_on_behavior(['off', 'on', 'previous']).withAccess(ea.STATE_SET),
        e.enum('indicator', ea.ALL, ['off', 'off/on', 'on']).withDescription('LED indicator mode').withAccess(ea.STATE_SET),
        e.numeric('fan_speed', ea.STATE_SET).withValueMin(1).withValueMax(5).withValueStep(1).withDescription('Speed of the fan'),
    ],
    meta: {
        tuyaDatapoints: [
            // Datapoints from Tuya IoT platform:
            //"1":"Fan switch",
            //"2":"Fan countdown",
            //"11":"Power-on state setting", enum. Off / On / Previous
            //"12":"Indicator status setting", enum. Off / (Off/On Status) / On
            //"101":"Fan Level" enum. 0 = Low, 1 = Medium, 2 = High            
            [1, 'state', tuya.valueConverter.onOff],
            [2, 'countdown', tuya.valueConverter.countdown],
	        [5, 'status_indication', tuya.valueConverter.onOff],
            [11, 'power_on_behavior', tuya.valueConverterBasic.lookup({'off': tuya.enum(0), 'on': tuya.enum(1), 'previous': tuya.enum(2)})],
            [12, 'indicator', tuya.valueConverterBasic.lookup({'off': tuya.enum(0), 'off/on': tuya.enum(1), 'on': tuya.enum(2)})],
            [101, 'fan_speed', tuya.valueConverterBasic.lookup({'1': tuya.enum(0), '2': tuya.enum(1), '3': tuya.enum(2), '4': tuya.enum(3), '5': tuya.enum(4)})],
        ],
    },
};

module.exports = definition;`

![interr](https://github.com/Koenkk/zigbee2mqtt/assets/149814514/c541c6e0-ea50-4135-b833-b9d2fe8bd4fa)
`

**Reply by flaviocorrea13** [reply]
Hello, were you able to solve the problem? I have two fans with the same problem.

**Comment by han-alink** [comment]
Hi,
As several of my DIY devices are using the "extend" functionality, just removing this line is not enough. As an example:

    {
    zigbeeModel: ['SprinkCtrl'], // The model ID.
    model: 'Controller for 5 sprinkler zones', 
    vendor: 'Alink',
    description: 'Device for the control of 5 sprinkler zones and a master valve',
    extend: extend.switch(),
    exposes: [e.switch().withEndpoint('l1'), e.switch().withEndpoint('l2'),
            e.switch().withEndpoint('l3'), e.switch().withEndpoint('l4'), e.switch().withEndpoint('l5'), e.switch().withEndpoint('l6')],
    meta: {multiEndpoint: true},
    endpoint: (device) => {
            return {l1: 1, l2: 2, l3: 3, l4: 4, l5: 5, l6: 6};    
            },
        configure: async (device, coordinatorEndpoint, logger) => {
            const overides = {min: 1, max: 3600};
            await reporting.bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
            await reporting.onOff(device.getEndpoint(1), overides);
            await reporting.bind(device.getEndpoint(2), coordinatorEndpoint, ['genOnOff']);
            await reporting.onOff(device.getEndpoint(2), overides);
            await reporting.bind(device.getEndpoint(3), coordinatorEndpoint, ['genOnOff']);
            await reporting.onOff(device.getEndpoint(3), overides);
            await reporting.bind(device.getEndpoint(4), coordinatorEndpoint, ['genOnOff']);
            await reporting.onOff(device.getEndpoint(4), overides);
            await reporting.bind(device.getEndpoint(5), coordinatorEndpoint, ['genOnOff']);
            await reporting.onOff(device.getEndpoint(5), overides);
            await reporting.bind(device.getEndpoint(6), coordinatorEndpoint, ['genOnOff']);
            await reporting.onOff(device.getEndpoint(6), overides);            
        },
    },

**Reply by mrskycriper** [reply]
If you need help, this is what the new converter will look like:

```js
const {deviceEndpoints, onOff} = require('zigbee-herdsman-converters/lib/modernExtend');

const definition = {
    zigbeeModel: ['SprinkCtrl'],
    model: 'Controller for 5 sprinkler zones',
    vendor: 'Alink',
    description: 'Device for the control of 5 sprinkler zones and a master valve',
    extend: [
        deviceEndpoints({endpoints: {'l1': 1, 'l2': 2, 'l3': 3, 'l4': 4, 'l5': 5, 'l6': 6}}),
        onOff({endpointNames: ['l1', 'l2', 'l3', 'l4', 'l5', 'l6']}),
    ],
};

module.exports = definition;
```

**Reply by han-alink** [reply]
Thanks!
But: How can I set reporting with my own max, min, delta values? Cannot find anything in the documentation.

**Reply by mrskycriper** [reply]
Custom intervals are supported for `onOff` extend right now, is there any particular reason to change them? It will report instantly with change anyway.

**Reply by han-alink** [reply]
I also set these intervals for battery operated devices:

    zigbeeModel: ['moistureDet'], 
    model: 'MoistureDetector', 
    vendor: 'Alink', 
    description: 'Moisture Detector/Sensor',
    fromZigbee: [fz.humidity, fz.battery],
    toZigbee: [],
    exposes: [e.humidity(), e.battery(), e.battery_voltage()],
    meta: {battery: {voltageToPercentage: '3V_2500_3000'}},
    configure: async (device, coordinatorEndpoint, logger) => {
        const endpoint1 = device.getEndpoint(1);
        await reporting.bind(endpoint1, coordinatorEndpoint, ['genPowerCfg']);
        await reporting.batteryVoltage(endpoint1);
        const overides = {min: 3600, max: 40000, delta: 20};
        const endpoint2 = device.getEndpoint(2);        
        await reporting.bind(endpoint2, coordinatorEndpoint, ['msRelativeHumidity']);
        await reporting.humidity(endpoint2, overides);

**Reply by mrskycriper** [reply]
Use this, custom reports for them are supported and make sense.

```js
const {humidity, battery} = require('zigbee-herdsman-converters/lib/modernExtend');

const definition = {
    zigbeeModel: ['moistureDet'],
    model: 'MoistureDetector',
    vendor: 'Alink',
    description: 'Moisture Detector/Sensor',
    extend: [
        humidity(reporting: {min: 3600, max: 40000, change: 20}),
        battery({{voltageToPercentage: '3V_2500_3000', voltage: true, voltageReporting: true, percentageReporting: false}}),
    ],
};

module.exports = definition;
```

**Reply by han-alink** [reply]
In reporting.ts the minimum reporting value  for the onOff attribute is set to 0. According to the home automation profile, this value should be a minimum  of 1.
(" The minimum reporting interval specified in [B2] shall be
set to a value greater than or equal to 0x0001.")
I'm not sure if this is still true for the latest release of this specification. My devices check on this value; the "0"  leads to an 'INVALID_VALUE' status.

**Reply by mrskycriper** [reply]
For `onOff` extend the minimum is set to 1

**Reply by han-alink** [reply]
I'm probably looking at the wrong lines.
I see the following:
await setupAttributes(device, coordinatorEndpoint, 'genOnOff', [{attribute: 'onOff', min: 'MIN', max: 'MAX', change: 1}]);
And MIN is set to 0.

**Reply by mrskycriper** [reply]
Thanks for pointing this out, it is indeed set to 0. I'll whatI can do about this.

**Reply by han-alink** [reply]
In the meantime I'm trying to set my own min and max values. As follows:

    zigbeeModel: ['OnOffOutput'],
    model: 'OnOffOutput',
    vendor: 'Alink           ',
    description: 'Light Controller',
    extend: [onOff(reporting: {min: 10, max: 40000})],

For some reason this leads to an error when starting zigbee2mqtt (syntax error)

**Reply by mrskycriper** [reply]
Custom report interval is simply not supported for onOff extend right now

**Reply by han-alink** [reply]
I took a look at other fixed reporting values.
Why should you use a reporting time of 10 seconds for f.i. a soilMoisture or humidity sensor? These devices are most times battery operated and transmitting values should be executed at a very low rate. 
A custom report interval for battery operated devices will be very usefull ... (or am I looking at the wrong place?)

**Reply by mrskycriper** [reply]
10 seconds is a minimum value, if there was a charge. And custom configuration is already supported for those kinds of modern extend

**Reply by han-alink** [reply]
Thanks for your help. I'm still struggling with reporting. What is wrong in this device description (custom reporting values for humidity):

    zigbeeModel: ['MultiSensTH'],
    model: 'MultiSensTH',
    vendor: 'Alink           ',
    description: 'Temperature and Humidity  device',
    extend: [deviceEndpoints({"endpoints":{"1":1,"2":2}}), temperature({"endpointNames":["1"]}), batteryPercentage(), 
    humidity({"endpointNames":["2"]}, reporting: {min: 300, max: 60000, change: 10})],
    meta: {"multiEndpoint":true}

**Reply by mrskycriper** [reply]
The `meta: {"multiEndpoint":true}` line is already added by `deviceEndpoints` modern extend.

Humidity `reporting` needs to be inside the same object as `endpointNames`

**Reply by han-alink** [reply]
Well, I have about 20 device descriptions in use for DIY devices. Updating to zigbee2mqtt release 1.36.1-1 caused almost all devices to stop working. So I reverted to 1.36.0-1 and am trying to update all device descriptions to use modernExtend. Thats is causing problems.
I changed the one I referenced in my last post. It is still not working. The temperature part (without custom reporting) works, but humidity fails.

    zigbeeModel: ['MultiSensTH'],
    model: 'MultiSensTH',
    vendor: 'Alink           ',
    description: 'Automatically generated definition',
    extend: [deviceEndpoints({"endpoints":{"1":1,"2":2}}), temperature({"endpointNames":["1"]}), batteryPercentage(), humidity({"endpointNames":["2"], reporting: {min: 300, max: 60000, change: 10}})]

**Reply by mrskycriper** [reply]
Can you please create a new issue and share all external definitions there?

**Reply by han-alink** [reply]
New thread: #22150

**Comment by sanderlv** [comment]
Got an instant error after updating... made me decide to completely remove the hue wall switch...

https://community.home-assistant.io/t/z2m-rdm001-not-working-anymore-after-update-to-1-36-1-1-929003017102/713862

And in the z2m documentation for the hue wall switch https://www.zigbee2mqtt.io/devices/929003017102.html there is an error... resetting works only when pressing 10 sec and close input TWO times.

**Comment by MOROHEUZ** [comment]
Unfortunately, I also had to open an issue for Hassio
[https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/issues/591](https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/issues/591)
the Bosch Twinguards no longer updated any values.
Even a reset / relearning had no effect.

I'm also not sure whether my system has frozen as a result.
Unfortunately, I had also installed the 2024.4.1 update from HA Core.
somehow 100s of requests went out to Alexa.
also a whole lot of climate issues which all had to do with the Twinguard.

I hope the problem can be solved.
I can't do without the Twinguards.

**Comment by sam3487** [comment]
Can we use the new ember adapter now without needing to run z2m through the dev branch?

**Reply by sam3487** [reply]
Anyone got an answer to this? Thanks :)

**Comment by array81** [comment]
There is a problem with the code relating to the IMOU siren, the official code is different from the one I proposed and with this it is not possible to make the siren sound via SET MQTT (you get an error). I talk about it [here](https://github.com/Koenkk/zigbee2mqtt/issues/21716).

Furthermore the siren does not have a battery so there is no point in exposing the battery.

**Comment by blakefrain** [comment]
This update seems to break commands to the deconz conbee 2
![image](https://github.com/Koenkk/zigbee2mqtt/assets/48569400/3768d1fe-b3c2-47f3-b8c4-4d9203ebabff)

**Reply by blakefrain** [reply]
see related issue https://github.com/Koenkk/zigbee2mqtt/issues/22189
